app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -586,14 +590,40 @@ public boolean onActionItemClicked(ActionMode mode, MenuItem item) {
             int nSelected;
             switch (item.getItemId()) {
                 case R.id.menu_item_delete:
-
-                    for (int i = getSelectedItems().size() - 1; i >= 0; i--) {
-                        deleteMessage(getSelectedItems().get(i));
-                    }
-                    Snackbar.make(recyclerView, "Deleted Messages Successfully!!", Snackbar.LENGTH_LONG).show();
-                    actionMode.finish();
+                    AlertDialog.Builder d = new AlertDialog.Builder(context);
+                    d.setMessage("Are you sure ?").
+                            setCancelable(false).
+                            setPositiveButton("Delete", new DialogInterface.OnClickListener() {
+                                @Override
+                                public void onClick(DialogInterface dialog, int which) {
+
+                                    for (int i = getSelectedItems().size() - 1; i >= 0; i--) {
+                                        deleteMessage(getSelectedItems().get(i));
+                                    }
+                                    Snackbar.make(recyclerView, "Deleted Messages Successfully!!", Snackbar.LENGTH_LONG).show();
```

Comment:
For the time being, you could put something as "Message(s)" maybe ?
Whatsapp shows a toast as "`x` messages deleted".
Any other suggestions ?